back/src/api/common/services/tobacco/tobaccoTaxCalculator.ts

```
@@ -0,0 +1,144 @@
+    },
+  };
+
+  private static readonly TYPE_MAPPING = {
+    cigarette: 'tobaccoCigarettes',
+    cigar: 'tobaccoCigars',
+    cigarillos: 'tobaccoCigarillos',
+    tobacco: 'tobaccoOther',
+  } as const;
+
+  private static mapType(type: string): string {
+    return this.TYPE_MAPPING[type as keyof typeof this.TYPE_MAPPING] || type;
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)